tests/cases/compiler/extendingCollectionsWithCheckJs.ts

```diff
@@ -3,6 +3,7 @@
 // @noEmit: true
 // @lib: es2017
 
+// @filename: extendingCollectionsWithCheckJs.js

```

Comment:
=(

Reply:
This has to end in `.js` so that the code is actually type checked as a **JavaScript** file.